src/App.js

```diff
@@ -49,8 +44,8 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
-               // Additional props (event callbacks, etc.) can be passed here
+               type={item.type}
+               score={this.onItemClicked}
```

Comment:
Score is not a great name for this, as it doesn't tell the reader that it is a callback, or even a function.